recipes/r-clusteval/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 conda-forge
```

Comment:
What did you try to do here?

Reply:
Do you mean the missing "(c)"? or using "conda-forge" in the license file?

Reply:
The MIT license is a template license, where the real upstream package author shall be filled in (which is most probably not conda forge). 

Normally it would be:
```
license_file: 
  - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
  - LICENSE
```
where the first line is the general license and the second line a file in the cran tarball that contains the author etc. That is not possible here, as the cran tarball does unfortunately not contain a file called LICENSE with the author.

Reply:
Got it. I originally was thinking to use the original author, but creating a license in someone else's name without their knowledge didn't seem right. Using "conda-forge" also didn't seem quite right, but a quick check in the existing feedstocks sugggests that that is the norm:

```
ag Copyright | grep conda-forge | wc -l
```

(that doesn't include all feedstocks; only a partial set..)

In any case, I should have just asked to be sure. Thanks for taking the time to clarify! Going to update recipes now.